src/hotspot/cpu/aarch64/globals_aarch64.hpp

```diff
@@ -127,6 +127,8 @@ define_pd_global(intx, InlineSmallCode,          1000);
           range(1, 99)                                                  \
   product(ccstr, UseBranchProtection, "none",                           \
           "Branch Protection to use: none, standard, pac-ret")          \
+  product(ccstr, UseSleefLib, "libsleef.so.3", EXPERIMENTAL,            \
+          "Sleef library to use for the vector math operations")        \
```

Comment:
Experimental functionality like this should not be enabled by default as you are changing the behaviour for all users. This needs to be off by default with user's being able to opt-in if they want.

Reply:
Sounds reasonable. Thanks a lot for the reminder!

Reply:
Hard-coding the libsleef ABI version into OpenJDK is a code smell. For now I suppose it'll do, but we need a better strategy going forward, perhaps involving a bundled library.

Reply:
Yes, that's true. I will consider it. Thanks for the advice!